python/cudf/dataframe/dataframe.py

```diff
@@ -805,30 +806,37 @@ def _sort_by(self, sorted_indices):
             df[k] = self[k].take(sorted_indices.to_gpu_array())
         return df
 
+    def argsort(self, ascending=True, na_position='last'):
+        cols = [series._column for series in self._cols.values()]
+        return get_sorted_inds(cols, ascending=ascending,
+                               na_position=na_position)
+
     def sort_index(self, ascending=True):
         """Sort by the index
         """
         return self._sort_by(self.index.argsort(ascending=ascending))
 
-    def sort_values(self, by, ascending=True):
+    def sort_values(self, by, ascending=True, na_position='last'):
         """
 
-        Uses parallel radixsort, which is a stable sort.
+        Sort by the values row-wise.
 
         Parameters
         ----------
         by : str
             Name of Series to sort by
-        ascending : bool, default True
-            Sort ascending vs. descending.
+        ascending : bool or list of bool, default True
+            If True, sort values in ascending order, otherwise descending.
+        na_position : str
+            first puts nulls at the beginning, last puts nulls at the end
```

Comment:
```suggestion
            'first' puts nulls at the beginning, 'last' puts nulls at the end
```

Since this is a `str` parameter and it's possible to read this line as a confusing sentence!